MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -1232,6 +1247,9 @@ - (BOOL) doubleValueIsEqual:(NSNumber *) x toOther:(NSNumber *) n {
 }
 
 - (void)didTriggerManualVideoChange {
+    // TODO: Should we deprecate? we no longer need this method. videoChange can happen whenever it's needed now, not just when AVPlayerItem changes
```

Comment:
Strong +1 to deprecate

Reply:
If keeping this functionality around makes sense we could also expose a better named API with the same functionality while deprecating this one. (And we'd drop the deprecated method next major version bump.)